testnets/axelartestnet/assetlist.json

```diff
@@ -346,10 +346,10 @@
           "counterparty": {
             "chain_name": "manifesttestnet",
             "base_denom": "umfx",
-            "channel_id": "channel-3"
+            "channel_id": "channel-6"
           },
           "chain": {
-            "channel_id": "channel-590",
+            "channel_id": "channel-594",
```

Comment:
with a new channel number, the `path` property must also be updated accordingly, and the `base` and `denom_unit` will also have a different hash now. This is the case for all assets affected by the change in channels